src/features/people/PeopleDetails/index.js

```diff
@@ -1,7 +1,45 @@
-function PersonDetails() {
+import React, { useEffect } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { useParams } from "react-router-dom";
+import {
+    selectStatus,
+    selectDetails,
+    getDetailsForPerson,
+} from "./peopleDetailsSlice";
+import { PersonDetailsTile } from "./PersonDetailsTile";
+
+const PersonDetails = () => {
+    const dispatch = useDispatch();
+    const { id } = useParams();
+
+    useEffect(() => {
+        dispatch(getDetailsForPerson({ personId: id }));
+        console.log("Dispatching getDetailsForPerson action");
```

Comment:
Do we need that log in deployed app?

Reply:
Thank you for catching that! 😀The console log was only for debugging purposes during the coding phase, and I forgot to remove it. It has no purpose in the deployed app, so I have now removed the unnecessary logs 😁
Appreciate your sharp eye for details! 😀